src/main/java/formats/AckMessage.java

```diff
@@ -32,7 +32,6 @@ public AckMessage(int blockNum)
 
     /**
      * @return A byte representation of the message
-     * @throws IOException
```

Comment:
Why take this out?

Reply:
Must have been done accidentally, will fix!